.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
```

Comment:
For ART managed content like `cloud-event-proxy` or `ptp-operator` this update is made through the ART bot.. which would not be the base with `BMER` or `hw-event-proxy` which are manged through CPaaS so we would have to update these manually

Reply:
ok, removed .ci-operator.yaml